### ISSUE TITLE
Firefox version?

### ISSUE THREAD

**Issue by fmasa** [martenbjork/github-xp#5]
Will you make Firefox version?

**Comment by Majkl578** [comment]
Already seems to work, to try it out:
1) clone this repo
2) package the contents into a zip file (so that manifest.json is in its root)
3) rename .zip to .xpi
4) go to about:debugging and load the extension from file

**Comment by fmasa** [comment]
@Majkl578 You're right, thanks! The only problem is Github icon being little off. I may look into it later.
![image](https://user-images.githubusercontent.com/5658260/41006794-4b9b92d8-6923-11e8-9351-1483b2980742.png)

**Comment by gaby-de-wilde** [comment]
perhaps: https://github.com/openstyles/stylus/wiki/Usercss

**Comment by gaby-de-wilde** [comment]
just pasting https://raw.githubusercontent.com/martenbjork/github-xp/master/xp.css into a new userstyle on https://addons.mozilla.org/en-US/firefox/addon/styl-us/ and setting it to domain github.com works.

**Comment by Krutonium** [comment]
I missed limiting it to GitHub. This CSS actually nearly works on Google, as well.

**Comment by vaniakosmos** [comment]
fixed Github icon problem with 

```
.header-logo-invertocat:before {
  content: "Github XP";
  font-style: italic;
  position: absolute;   /* this */
  top: 19px;   /* this */
  left: 50px;   /* this */
  font-weight: bold;
}
```

**Comment by vaniakosmos** [comment]
also you don't need to pack .xpi file, just point firefox to manifest.json

**Comment by BlackoutWNCT** [comment]
I made a paste with the icon fix in it, confirmed working.

https://paste.ubuntu.com/p/2hNWH7yZqb/

Paste this into a new style, add domain, easy done.
